src/app/domains/views/DomainDetails/ResourceRecords/DeleteRecordForm/DeleteRecordForm.tsx

```diff
@@ -24,10 +24,10 @@ export const Labels = {
 } as const;
 
 const DeleteRecordForm = ({
-  closeForm,
   id,
   resource,
 }: Props): React.ReactElement | null => {
```

Comment:
I generally convert these for better typing, but it's not a requirement.
```suggestion
}: Props): ReactElement => {
```